### PR TITLE
Switch to storybook

### PR DESCRIPTION
**Pull request by goatslacker**
Switching from cosmos-js to storybook and upgrading babel to 6 since that's what we're now using.

@ljharb @lencioni

### CODE REVIEW

File: stories/ExampleSlider.jsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      }}>
+        <Rheostat
+          {...this.props}
+          onValuesUpdated={this.updateValue}
+          values={this.state.values}
+        />
+        <ol>
+          <lh>Values</lh>
+          {this.state.values.map((value, i) => {
+            return (
```
Comment (lencioni) [inline stories/ExampleSlider.jsx]: This is a redundant return. You can remove it and change the curlies to parens.

File: stories/ExampleSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+              <li key={i}>
```
Comment (lencioni) [inline stories/ExampleSlider.jsx]: It would be best to find a better key for this than the positional index.

File: stories/ExampleSlider.jsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        values={[0, 100]}
+      />
+    );
+  })
+  .add('Dates', () => {
+    const startDate = new Date('01-01-2015').valueOf();
+    const endDate = new Date('12-31-2015').valueOf();
+    const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+
+    function num(n) {
```
Comment (lencioni) [inline stories/ExampleSlider.jsx]: nit: I think a better name for this function would be `ordinal`

**Comment by lencioni** [comment]
LGTM

File: stories/ExampleSlider.jsx (hunk 1 above)
Comment (ljharb) [inline stories/ExampleSlider.jsx]: does the linter not require this here?

Comment (lencioni) [inline stories/ExampleSlider.jsx]: I think it's on old versions, so updating would probably find this one.

**Pull request merged**